lib/python/picongpu/input/parameters.py

```diff
@@ -185,13 +210,51 @@ def on_value_change(self, change):
         if change["type"] == "change":
             return self.set_value(x=change["new"])
 
-    def get_value(self):
+    def convert_value_from_internal_scale(self, x):
+        """
+        Used for value conversion from the parameters internal scale
+        to the ui scale
+        It computes the inverse of the transformation
+        that is used in convert_value_to_internal_scale().
+
+        Since those scales are the same for UiParameter
+        objects, this computes the identity function.
+
+        In derived classes however, there might exist a
+        transformation function to convert between ui scale
+        values and internal parameter scale values
+        (e.g. slider shows values between
+        3 and 6, but the internal parameter scale is
+        between 3.e-9 and 6.e-9).
+
+        Parameters
+        ----------
+        x: float
+            a value on the same scale as the parameters self.value
+
+        Returns
+        -------
+        the outcome of the identity function since the ui
+        scale and the internal scale are identical.
+        """
+
+        return x
+
+    def get_value_on_ui_scale(self):
         """
+        Function that reverts a possible transformation of ui
+        value to internal value.
+        Since for UiParameter there is no conversion, it returns
+        the self.value.
+        In derived classes however, there will be a reverse trans-
```

Comment:
please avoid manual line-breaks since documentation generators will then "gen-erate" the same hyphen.
Just put the word on the next line :)

Reply:
Ok, I fixed that.